packages/terra-site/src/examples/content-container/ContentContainerStandard.jsx

```diff
@@ -23,9 +24,10 @@ pharetra, augue felis aliquet nunc, non facilisis est velit vel elit.
                  </p>);
 
 const contentheader = <Heading level={1} size={'large'}>Display Header</Heading>;
+const contentfooter = <Button text="Save" />;
```

Comment:
Nit: For consistency and clarity in the examples, can we rename the footer text to 'Display Footer'?